src/index.html

```diff
@@ -6,9 +6,78 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a
+          href="#home"
+          class="logo__link"
+        >
+          <img
+            src="./images/logo.png"
+            alt="MOYO logo"
+            class="logo__img">
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >Apple</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >Samsung</a>
```

Comment:
if you have only 2 attributes you don't need move tham to a new line
```suggestion
            <a href="#samsung" class="nav__link">Samsung</a>
```

Reply:
✅ Fixed

But reason of PR reject is unknown - critical/bad practice/bad readability/guidelines?

I've read MA code guidelines and it says wrapping [**is optional**](https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping).
I understand that we need to keep our code readable and follow guidelines. For me this html line:
```
<a
  href="#samsung"
  class="nav__link"
>Samsung</a>
```
is much more readable and easier to edit than
```
<a href="#samsung" class="nav__link">Samsung</a>
```

Pros:
- separation between tagname and its attributes is more obvious;
- easy to add/remove attributes if necessary (even with only two ones);
- avoiding line overflow over 80 symbols in case of long values (class and etc.)

Cons:
- Much longer html code

Reply:
@sergii-nosachenko 

  Your arguments are quite valid, both on the ease of adding new attributes and on readability. But in future homeworks, Lint will complain more often about this kind of transposition.
  You should also be aware that when you start working on a new project in a new company, you may encounter new architecture and lint rules, and you will have to abide by them. We are commenting and asking you to follow the checklists and tasks so that you don't make mistakes from the beginning. And those mistakes aren't become a habit. I'll approve this work cause here that rule is really optional.

Reply:
@Andriyrut Thanks, I understand. Don't be angry on me, you comment are keeping me focused and I double-check my code now :)

Reply:
@sergii-nosachenko 
I don't have any reason to be angry! Sorry if I made or write something that makes think you like that))